File: 프로그래머스/기초 트레이닝 - Level 0/문자 개수 세기-ex.py
def solution(my_string):
    answer = [0] * 52
    for alpha in my_string:
        if alpha.isupper():
            answer[ord(alpha) - ord('A')] += 1
        else:
            answer[(ord('Z') - ord('A') + 1) + ord(alpha) - ord('a')]+= 1
    return answer

File: 프로그래머스/기초 트레이닝 - Level 0/문자 개수 세기-ex.java
class Solution {
    public int[] solution(String my_string) {
        int arrLength = (int) 'Z' - (int) 'A' + (int) 'z' - (int) 'a' + 2;
        int[] answer = new int[arrLength];
        int length = my_string.length();
        
        for (int i = 0; i < length; i++) {
            char c = my_string.charAt(i);
            if (c >= 'a') 
                answer[c - 'a' + 'Z' - 'A' + 1]++;
            else 
                answer[c - 'A']++;
        }
        
        return answer;
    }
}